mock-core-configs/etc/mock/templates/fedora-eln.tpl

```diff
@@ -53,11 +48,10 @@ file:///usr/share/distribution-gpg-keys/fedora/RPM-GPG-KEY-fedora-rawhide-primar
 {%- endfor %}
 {%- endmacro %}
 
-{% if mirrored %}
 [eln-baseos]
 name=Fedora - ELN BaseOS - Developmental packages for the next Enterprise Linux release
-baseurl=https://odcs.fedoraproject.org/composes/production/latest-Fedora-ELN/compose/BaseOS/$basearch/os/
-#metalink=https://mirrors.fedoraproject.org/metalink?repo=eln&arch=$basearch
+#baseurl=https://dl.fedoraproject.org/pub/eln/1/BaseOS/$basearch/os/
+metalink=https://mirrors.fedoraproject.org/metalink?repo=eln-baseos-1&arch=$basearch
```

Comment:
Why the `-1` suffix?  Are we going to bump in the future?  How often?  Do we want to have a variable for this?

Reply:
That's a limitation of mirror manager. It has no actual meaning and should never change.

Mirror manager just can't cope without a "version" (and adapting every place that it special-cases "rawhide" was prohibitively difficult). So we just decided it would be version "1" forever.